.gitignore

```diff
@@ -16,4 +16,5 @@ AssemblyVersionInfo.fs
 miTLS/
 Backup/
 paket.exe
-!paket.lock
\ No newline at end of file
+!paket.lock
+**.suo
```

Comment:
a single star doesn't check subdirectories, and the file will usually be in /src/